packages/components/src/DatePicker/DatePicker.tsx

```
@@ -83,7 +83,13 @@ interface DatePickerInlineProps extends BaseDatePickerProps {
 
 type DatePickerProps = XOR<DatePickerModalProps, DatePickerInlineProps>;
 
-/*eslint max-statements: ["error", 13]*/
+interface DatePickerCloseDetails {
+  id: string;
+}
+
+export const DATEPICKER_SWITCH_EVENT = "atlantis.datepicker-switch";
+
+/*eslint max-statements: ["error", 16]*/
```

I think it would be good to pull some things out of the function rather than silencing the error

---

we've gone with a totally different approach and it's really now just a 1 line change in this file, so will respectfully not do this until we have to refactor this component in the future 😆